bitnami/etcd/README.md

```diff
@@ -219,24 +216,33 @@ Apart from providing your custom configuration file, you can also modify the ser
-| `ETCD_CONF_DIR`             | etcd configuration directory.                                        | `${ETCD_BASE_DIR}/conf`            |
-| `ETCD_DEFAULT_CONF_DIR`     | etcd default configuration directory.                                | `${ETCD_BASE_DIR}/conf.default`    |
-| `ETCD_TMP_DIR`              | Directory where ETCD temporary files are stored.                     | `${ETCD_BASE_DIR}/tmp`             |
-| `ETCD_CONF_FILE`            | ETCD configuration file.                                             | `${ETCD_CONF_DIR}/etcd.yaml`       |
-| `ETCD_NEW_MEMBERS_ENV_FILE` | File containining the etcd environment to use after adding a member. | `${ETCD_DATA_DIR}/new_member_envs` |
-| `ETCD_DAEMON_USER`          | etcd system user name.                                               | `etcd`                             |
-| `ETCD_DAEMON_GROUP`         | etcd system user group.                                              | `etcd`                             |
+| Name                         | Description                                                          | Value                              |
+|------------------------------|----------------------------------------------------------------------|------------------------------------|
+| `ETCD_BASE_DIR`              | etcd installation directory.                                         | `/opt/bitnami/etcd`                |
+| `ETCD_VOLUME_DIR`            | Persistence base directory.                                          | `/bitnami/etcd`                    |
+| `ETCD_BIN_DIR`               | etcd executables directory.                                          | `${ETCD_BASE_DIR}/bin`             |
+| `ETCD_DATA_DIR`              | etcd data directory.                                                 | `${ETCD_VOLUME_DIR}/data`          |
+| `ETCD_CONF_DIR`              | etcd configuration directory.                                        | `${ETCD_BASE_DIR}/conf`            |
+| `ETCD_DEFAULT_CONF_DIR`      | etcd default configuration directory.                                | `${ETCD_BASE_DIR}/conf.default`    |
+| `ETCD_TMP_DIR`               | Directory where ETCD temporary files are stored.                     | `${ETCD_BASE_DIR}/tmp`             |
+| `ETCD_CONF_FILE`             | ETCD configuration file.                                             | `${ETCD_CONF_DIR}/etcd.yaml`       |
+| `ETCD_NEW_MEMBERS_ENV_FILE`  | File containining the etcd environment to use after adding a member. | `${ETCD_DATA_DIR}/new_member_envs` |
+| `ETCD_DAEMON_USER`           | etcd system user name.                                               | `etcd`                             |
+| `ETCD_DAEMON_GROUP`          | etcd system user group.                                              | `etcd`                             |
+| `ETCD_INITIAL_CLUSTER_STATE` | Initial cluster state. Either "new" or "existing".                   | `nil`                              |
```

Comment:
Is this still used?

Reply:
The user does not use it. Is it fine to leave it in the read-only section then?

Reply:
Removed